azavea/raster-foundry
# Feature/ak/add datasource model

## Overview

Datasources have more attributes than names, for example, default color corrections and visibility. To track information about datasources we needed a datasource model.

This PR adds a datasource table to the database, adds a foreign key constraint from scenes to datasources, adds datasources for Landsat 8 and Sentinel-2 to the database, and uses those datasource IDs in the airflow importers.

### Checklist

- ~[x] Swagger specification updated, if necessary~ for mysterious reasons, the [swagger spec already said](https://github.com/azavea/raster-foundry/blob/develop/docs/swagger/spec.yml#L1085-L1093) that the datasource was a UUID-formatted string.

## Notes

This breaks the front-end datasource filter.

## Testing Instructions

- `app/test` from your sbt console
- `./scripts/server --with-airflow`, wait a while (you may want to do this at the end of the day and then just leave it), verify that that you have more scenes the next morning than you did when you left
